docs/admin/ServerSideKeygen.adoc

```diff
@@ -0,0 +1,157 @@
+= ServerSide Key Generation for EE
+Author <cfu@redhat.com>
+:toc:
+:imagesdir: images
+
+== Overview
+
+Many newer versions of browsers, including Firefox v69 and up, as well as Chrome, have removed the functionality to generate PKI keys and the support for CRMF for key archival. While CLIs such as CRMFPopClient (see "CRMFPopClient --help") or pki (see "pki client-cert-request --help") could be used as a workaround on the Fedora or RHEL platforms, clients on non-RedHat platforms have been largely left out. 
```

Comment:
Red Hat should be 2 words :)
or use "... on other platforms ..."

Reply:
sounds good!